src/components/index.js

```diff
@@ -333,12 +339,13 @@ export default class Rnd extends Component {
         onDrag={this.onDrag}
         onStop={this.onDragStop}
         axis={this.props.dragAxis}
+        disabled={this.props.disableDragging}
```

Comment:
I had to run `npm run prepublish` in order to `npm run flow` to get latest changes in the check. As a feedback for that contrib guide 😄